docs/user_guide.md

```diff
@@ -24,80 +24,92 @@ $ java -jar RepoSense.jar -config CSV_path.csv -output output_path/ -since 01/10
 Argument List:
 - config: Mandatory. The path to the CSV config file.
 - output: Optional. The path to the dashboard generated. If not provided, it will be generated in the current directory.
-- since : Optional. start date of analysis. Format: dd/MM/yyyy
-- until : Optional. end date of analysis. Format: dd/MM/yyyy
+- since : Optional. start date of analysis. Format: `DD/MM/YYYY`
+- until : Optional. end date of analysis. Format: `DD/MM/YYYY`
 
 ```
 Note:
 The contribution calculation is base on the daily commits made within 00:00 to 23:59 in GMT+8.
 ```
 
 ## CSV Config File
-The CSV Config files control the list of target repositories. It also contains a white list of authors(if the author is listed in the CSV, his/her contribution will be ignored by the analyzer.)
-[Sample_full.csv](../sample_full.csv) in root is an example CSV config file. It should contain the following columns:
+The CSV Config files control the list of repositories and the target author.
```

Comment:
CSV Config files -> CSV Config file

Currently, we are only supporting a file and also the header is singular.